src/Visualizer/hooks.ts

```diff
@@ -82,12 +82,24 @@ export default function useHooks(
       reason: LayerSelectionReason | undefined,
       info: SelectedFeatureInfo | undefined,
     ) => {
-      if (selectedLayer.layerId === layerId && selectedLayer.featureId === featureId) return;
+      const isSketchLayer =
+        selectedLayer.layer?.layer.type === "simple" &&
+        selectedLayer.layer?.layer?.data?.isSketchLayer;
+      // Sketch layer feature has a fixed featureId, we need to exclude it from the skip condition
+      if (
+        selectedLayer.layerId === layerId &&
+        selectedLayer.featureId === featureId &&
+        !isSketchLayer
+      )
```

Comment:
What does this mean? I thought you don't need to skip this condition if the id is fixed. Do you mean that you need to re-select the feature even if the selected id isn't changed?

Reply:
If this is skipped, i can only get the old value of computedFeature...

Reply:
Hmm I see. I understood.